controller:
  hero_chassis_controller:
    type: hero_chassis_controller/HeroChassisController
    Wheel_Track: 0.5
    Wheel_Base: 0.5
    Wheel_Radius: 0.05

  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50